src/main/java/seedu/duke/food/FoodList.java

```diff
@@ -0,0 +1,79 @@
+package seedu.duke.food;
+
+import java.util.ArrayList;
+
+public class FoodList {
+    private ArrayList<Food> foodRecords = new ArrayList<>();
+
+    /**
+     * Returns food item in the food list.
+     *
+     * @param index The index of the food item
+     * @return the food item with the given index
+     */
+    public Food get(int index) {
+        return this.foodRecords.get(index);
+    }
+
+    /**
+     * Returns the size of the array list.
+     *
+     * @return the size of the array list attribute
+     */
+    public int size() {
+        return foodRecords.size();
+    }
+
+    /**
+     * Adds a food item into the food list.
+     *
+     * @param food The food class object to add
+     */
+    public void add(Food food) {
```

Comment:
Would it be better to name this method differently from the other add method below? Since they would be parsed differently as well, I think it might be better to name them differently as there is no need to overload this method, and it is a bit confusing currently.

Reply:
The reason why I overload this add method is because I do not know which one you prefer to implement, will delete the redundant one once we merge

Reply:
Ohh that makes sense! I think I will be keeping this one, but yea we can keep both and remove the redundant one once we merge! Thanks for clarifying 👍

Reply:
my addExercise method is similar to this one, perhaps we can just use this one :)